### PR TITLE
Monday exercises

### PR DESCRIPTION
**Pull request by VitorGuedesMadeira**
### Vet Clinic - Database
- Use template for repo ✔️ 
- Create databse vet_clinic ✔️
- Create animals table ✔️
- Insert Data ✔️

screenshots:
Monday_1
Monday_2

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- MONDAY
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-01-01';
```
Comment (omar-muhamad) [inline queries.sql]: - [ ] in this query you suppose to include the whole year of 2019, not the first day in it only. So please edit this query.
[Hint] You can do it like that:
```sql
SELECT name FROM animals WHERE date_of_birth BETWEEN '2016/01/01' and '2019/31/12'
```
or
```sql
SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019
```

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @VitorGuedesMadeira,

Nice job, you are almost there.



## (Changes Required) Aspects to improve: ♻️

- Please edit the following in your readme file to make it more professional:
- [ ] The project name suppose to be something like `Vet Clinic Database` so please fix that.
- [ ] Please add a proper project description showing what you did and will do in this project.
- [ ] In the `Authors` section, Please add your name and social links.

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (VitorGuedesMadeira) [reply queries.sql]: Changed! ✔️ Thanks!

**Review by BertrandConxy — APPROVED** [review]
Hi @VitorGuedesMadeira, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 

- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**